### PR TITLE
 Merged Mahnoor's, Jaideep's, Reem's and my contribution

### PR DESCRIPTION
**Pull request by pollchowdhury**
Merging Reem's forum feature.

### CODE REVIEW

File: src/InsightsBrakes.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import React from 'react';
+
+const contentStyle = {
+  height: '160px',
+  color: '#fff',
+  lineHeight: '160px',
+  textAlign: 'center',
+  background: '#364d79',
+};
+
```
Comment (andreacuc) [inline src/InsightsBrakes.js]: implement clear and consistent comments throughout code

File: src/InsightsBrakes.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+const App = () => (
+  <Carousel autoplay>
+
+    <div style={contentStyle}>
+        <section>
+            <h1> Ceramic Brake Pads</h1>
```
Comment (andreacuc) [inline src/InsightsBrakes.js]: try implementing bootstrapping tags to help both with documentation and clearly group related code

File: src/Components/Forum/ForumPages/ForumHome.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    const postDoc = doc(db, "posts", id)
+    await deleteDoc(postDoc)
+  }
+
+  return (
+    <div className="homePage">
+        {/* //post variable has id for list doc */}
+      {postLists.map((post) => {
+        return (
+          <div className="post">
```
Comment (andreacuc) [inline src/Components/Forum/ForumPages/ForumHome.js]: organize the structure of the return statement/ incorporate more divs and comments to clearly establish the structure of the forum page

**Comment by jchunduri35** [comment]
Reem's Forum feature merged.

**Review by andreacuc — COMMENTED** [review]
code-review

**Pull request merged**